src/controllers/signup.js

```diff
@@ -0,0 +1,45 @@
+const addUser = require('../database/queries/addUser');
+const checkUser = require('../database/queries/checkUser');
+const hashpassword = require('../utillity/hashpassword');
+
+exports.get = (req, res) => {
+  res.render('signup', { js: 'signup' });
+};
+
+exports.post = (req, response) => {
+  const data = req.body;
+  const name = data.name;
+  const email = data.email;
+  const pass = data.pass;
+  const location = data.location;
+  const spec = data.spec;
+  const occupation = data.occupation;
+  const linkedin = data.linkedin;
+  const facebook = data.facebook;
+  const mobile = data.mobile;
+  const img = data.img;
+  const description = data.description;
```

Comment:
please use Destructuring variables
from line 11 to line 21 you can use this instead
``` const { name, email, pass, location, spec, occupation, linkedin, facebook, mobile, img, description } = data; ```